streams/src/main/java/org/apache/kafka/streams/state/internals/buffer/InMemoryTimeOrderedKeyValueBuffer.java

```diff
@@ -14,14 +14,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.kafka.streams.kstream.internals.suppress;
+package org.apache.kafka.streams.state.internals.buffer;
```

Comment:
Since the buffer is a proper state store now, I've moved it to the `state` package. I gave it a sub-package `buffer` to help contain its package-private members.

Reply:
I am not a fan of too many tiny sub-package (ie, don't create `buffer`?). But it's very subjective...

Reply:
I agree not to organize for the sake of organizing, but in this case, there are several subsidiary classes that really don't need to be used outside of this context, and there's no other way to hide them than to create this package.

I should say, other than putting them all inside the main Buffer class, but then it'll be pretty huge, which seems like it hurts readability.

Reply:
Ok, after taking some of your other advice, it was more manageable to pack all the private stuff into one class, so I've removed this package.